games/data/tcg-card-shop-simulator.yml

```diff
@@ -0,0 +1,58 @@
+uuid: "c49c7c87-81de-4877-aeb5-1793a88997e6"
```

Comment:
This seems to be an exact match with GoreBox. A game needs an unique uuid. Grab one e.g. from https://www.uuidgenerator.net/version4

Reply:
Is there a way for me to generate it while making sure it doesnt match pre existing ones and ill try and fix this around 4-5pm EST ^^

Reply:
Just use a random generated one and it shouldn't be a problem.

> The number of random version-4 UUIDs which need to be generated in order to have a 50% probability of at least one collision is 2.71 quintillion. This number is equivalent to generating 1 billion UUIDs per second for about 86 years.

Reply:
Understood, just wanted to make sure ^^

Reply:
Hopefully resolved